src/sections/Home/Layer5-statement/statement.style.js

```diff
@@ -47,14 +52,13 @@ const BannerSectionWrapper = styled.section`
         color: rgba(255, 255, 255, 0.7);
     }
     .hero {
-        background: url(${imgHero});
```

Comment:
Now I get why we were seeing the JPG being downloaded twice in the networks stats earlier 😶

Reply:
I had the same thought. :P I wonder what we were thinking at the time.